tools/src/wyvern/tools/typedAST/core/declarations/ModuleDeclaration.java

```diff
@@ -41,9 +41,10 @@
 
     private final List<ImportDeclaration> imports;
     private final List<GenericParameter> generics;
+    private final boolean isLifted;
```

Comment:
This is also a problem.  A module is not "lifted" or not, so it doesn't make sense to have a field here.  Instead, a module is lifted a certain import sites -- but not lifted at others.

This may be a problem with the original lifting code from this summer that is just being carried over.  But now that we are incorporating "lifted" into the import statement and not the module declaration, we should get it right.